electron/launcher.ts

```diff
@@ -259,7 +259,7 @@ function setupWineEnvVars(gameSettings: GameSettings, gameId = '0') {
   if (gameSettings.enableResizableBar) {
     ret.VKD3D_CONFIG = 'upload_hvv'
   }
-  if (gameSettings.useSteamRuntime) {
+  if (wineVersion.type === 'proton') {
```

Comment:
Ah one question:
dont we need that for the steam runtime for native games?

Reply:
No, these settings are only relevant for Proton
The function (`setupWineEnvVars`) shouldn't even be called for native games

Reply:
I see 👍🏽